aten/src/ATen/native/mps/operations/ConstantOps.mm

```diff
@@ -99,7 +114,14 @@ static bool fill_mps_tensor_(Tensor& self, uint8_t value) {
   Scalar scalar_value = value.item();
   if (scalar_value.toDouble() == 0.0 && fill_mps_tensor_(self, 0) == true)
     return self;
-  return fill_scalar_mps_impl(self, scalar_value);
+  return fill_scalar_mps(self, scalar_value);
+}
+
+Tensor& zero_mps_(Tensor& self) {
+  // check if it's possible to use fillBuffer() to fill the Tensor's storage
+  if (fill_mps_tensor_(self, 0) == true)
+    return self;
```

Comment:
No need to add it here, as you've already moved it to  `fill_scalar_mps`, aren't you?

Reply:
Oops, yes.